### ISSUE TITLE
possible audio fix?

### ISSUE THREAD

**Issue by kobykk** [burnedpopcorn/Pizza-Tower-1.1.0-Web-Port#11]
i found a error symbol next to this line of code new: AudioContext; also i found it by looking through the game with dev tools

**Comment by burnedpopcorn** [comment]
Yeah, thanks for the effort, but I think this is more of a Gamemaker extension issue. The game uses an extension called FMOD, but because extensions aren't supported when exporting to gx.games, FMOD isn't initiated

**Comment by burnedpopcorn** [comment]
That or the .bank files can't be read because anything other than the base game files are ignored and unreadable, even if they are in the same server

**Comment by burnedpopcorn** [comment]
My idea is that I manually rip the music files from the .bank files and manually rewrite every fmod function to instead point to their respective music file

**Comment by burnedpopcorn** [comment]
I am certain this would work, but it would take a lot of my time, plus it would add even more data to the files, which could complicate things with my web hosting service

**Comment by burnedpopcorn** [comment]
I am also certain because my April 2022 port of the game didn't use fmod at all, and that port actually have music, so I could just inject that code to this port

**Comment by kobykk** [comment]
> I am also certain because my April 2022 port of the game didn't use fmod at all, and that port actually have music, so I could just inject that code to this port

try that
